### ISSUE TITLE
Error connecting remote node

### ISSUE THREAD

**Issue by vdo** [mimblewimble/grin-wallet#191]
I have a remote node running in my local network. When I try to use grin wallet with it, I get an error, although the port is open, the API reachable, and actually the wallet commands end succesfully:

```
~/src/grin-wallet
 Ѫ ./grin-wallet -V                                
grin-wallet 2.0.0-beta.2

~/src/grin-wallet
 Ѫ curl "http://172.33.0.3:3413"                 
[
  "get blocks",
  "get headers",
  "get chain",
  "post chain/compact",
  "get chain/validate",
  "get chain/outputs/byids?id=xxx,yyy,zzz",
  "get chain/outputs/byheight?start_height=101&end_height=200",
  "get status",
  "get txhashset/roots",
  "get txhashset/lastoutputs?n=10",
  "get txhashset/lastrangeproofs",
  "get txhashset/lastkernels",
  "get txhashset/outputs?start_index=1&max=100",
  "get txhashset/merkleproof?n=1",
  "get pool",
  "post pool/push_tx",
  "post peers/a.b.c.d:p/ban",
  "post peers/a.b.c.d:p/unban",
  "get peers/all",
  "get peers/connected",
  "get peers/a.b.c.d",
  "get version"
]
~/src/grin-wallet
 Ѫ ./grin-wallet -r "http://172.33.0.3:3413" info
20190718 10:26:19.026 ERROR grin_wallet_impls::node_clients::http - Unable to contact Node to get version info: Request error: Cannot make request: an error occurred trying to connect: Connection refused (os error 111)
Password: 

____ Wallet Summary Info - Account 'default' as of height 263444 ____

 Confirmed Total                  | 0.000000000 
 Awaiting Confirmation (< 10)     | 0.000000000 
 Awaiting Finalization            | 0.000000000 
 Locked by previous transaction   | 0.000000000 
 -------------------------------- | ------------- 
 Currently Spendable              | 0.000000000 

Command 'info' completed successfully
```
daemon v2.0.0
wallet 2.0.0-beta.2

**Comment by yeastplume** [comment]
have you checked that the `.api_secret` is the same on your node and wallet?

**Comment by vdo** [comment]
No, I haven't.
This is step is missing in the wiki I'm afraid. 
I will check and try again, thanks

**Comment by michaeldzjap** [comment]
I am running into the same problem. I can confirm that this also happens when using the same `.api_secret` for both node and wallet.

**Comment by yeastplume** [comment]
This isn't a widespread issue, is this still occurring for you or have you traced down the problem?

**Comment by yeastplume** [comment]
Closed for lack of responses, note v2.1.0 onwards fixed issues relating to having a single .api_secret file
